File: app/src/main/java/com/levibostian/androidblanky/AppConstants.java
package com.levibostian.androidblanky;

public class AppConstants {

    public static String baseWebsiteHost = "http://curiosityio.com";

    public static String getPrivacyPolicyUrl() {
        return baseWebsiteHost.concat("/privacy_policy.html");
    }

    public static String getTermsAndConditionsUrl() {
        return baseWebsiteHost.concat("/terms_and_conditions.html");
    }

    public static String getAboutUrl() {
        return baseWebsiteHost.concat("/about_us.html");
    }

}

File: app/src/main/java/com/levibostian/androidblanky/AppConstants.kt
package com.levibostian.androidblanky

object AppConstants {

    var API_ENDPOINT = "https://api.github.com"

    var baseWebsiteHost = "https://curiosityio.com"

    val privacyPolicyUrl: String
        get() = baseWebsiteHost + "/privacy_policy.html"

    val termsAndConditionsUrl: String
        get() = baseWebsiteHost + "/terms_and_conditions.html"

    val aboutUrl: String
        get() = baseWebsiteHost + "/about_us.html"

}
